the-events-calendar.php

```diff
@@ -28,7 +28,9 @@
  * along with this program; if not, write to the Free Software
  * Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
  */
-define( 'TRIBE_EVENTS_FILE', __FILE__ );
+define('TRIBE_EVENTS_FILE', __FILE__);
+
+if(1 == __FILE__      ) $x = 5;
```

Comment:
🚫 **[PHP_CodeSniffer]** <TribalScents.Whitespace.VarSpacing.invalidWhitespace> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>There must be a space between if and the first parenthesis '('